src/main.rs

```diff
@@ -292,6 +297,9 @@ struct Opt {
     /// No symbol demangling.
     #[arg(long)]
     no_demangle: bool,
+    /// Print a summary of the results
+    #[arg(long)]
+    print_summary: bool,
```

Comment:
What if we have "summary" as one of the possible output types instead of adding a --print-summary option?